content/blog/2018/custom-gateway/index.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Add a new Custom IngressGateway
+description: Describes how to add a new custom ingress gateway manually.
+publishdate: 2018-10-26
+subtitle: Custom Ingress Gateway
+attribution: Julien SENON
+weight: 89
+keywords: [ingress,traffic-management, aws]
+---
+
+This post provide instructions to create manually a new custom ingress gateway with automatic provisioning of certificate based on cert-manager.
+
+This post was validated on AWS Infrastructure
+
+## Prerequisites
+
+Have cert-manager installed
+
+## Cert-Manager Configuration
+
+### Check installation
+
```

Comment:
is this a specific cert mgr helm chart?  can you put a convenience link?  Also, does this only work with AWS?  would be good to have a blog relatively general but the steps are for a given cloud.

Reply:
Thanks, I've push new version with your remarks. It will work with another cloud provider but special configuration is needed for cert-manager ACME validation with route53, and network loadblancer nlb that is cloud provider specific. I've set some warning on top of the configuration file. Let me know if it's better.